pkg/controller/disruption/disruption_test.go

```diff
@@ -556,6 +558,7 @@ func TestNakedPod(t *testing.T) {
 	dc.sync(ctx, pdbName)
 
 	ps.VerifyDisruptionAllowed(t, pdbName, 0)
+	verifyUnsupportedControllerEventEmitted(t, dc)
```

Comment:
Can you please add this check to `TestIntegerMaxUnavailable` as well? It would also be good to add a new test similar to these here, but the pod should have an unsupported controller owner reference.